server/sonar-server/src/main/java/org/sonar/server/usergroups/ws/CreateAction.java

```diff
@@ -103,7 +103,7 @@ public void handle(Request request, Response response) throws Exception {
   }
 
   private void writeResponse(Request request, Response response, OrganizationDto organization, GroupDto group) {
-    WsUserGroups.CreateResponse.Builder respBuilder = WsUserGroups.CreateResponse.newBuilder();
+    WsUserGroups.CreateWsResponse.Builder respBuilder = WsUserGroups.CreateWsResponse.newBuilder();
```

Comment:
remark: I had the feedback that we should call this classes `CreateResponse` without Ws. I don't have a strong opinion on this…